apps/sailfish/qml/pages/ResumeDialog.qml

```diff
@@ -0,0 +1,38 @@
+/*****************************************************************************
+ * Copyright: 2011-2013 Michael Zanetti <michael_zanetti@gmx.net>            *
+ *            2014      Robert Meijers <robert.meijers@gmail.com>            *
+ *                                                                           *
+ * This file is part of Kodimote                                           *
+ *                                                                           *
+ * Kodimote is free software: you can redistribute it and/or modify        *
+ * it under the terms of the GNU General Public License as published by      *
+ * the Free Software Foundation, either version 3 of the License, or         *
+ * (at your option) any later version.                                       *
+ *                                                                           *
+ * Kodimote is distributed in the hope that it will be useful,             *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of            *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the             *
+ * GNU General Public License for more details.                              *
+ *                                                                           *
+ * You should have received a copy of the GNU General Public License         *
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.     *
+ *                                                                           *
+ ****************************************************************************/
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Dialog {
+    property variant item
+
+    DialogHeader {
+        id: header
+        acceptText: qsTr("Resume")
+    }
+
+    Label {
+        anchors { left: parent.left; right: parent.right; top: header.bottom; leftMargin: Theme.paddingLarge; rightMargin: Theme.paddingLarge }
+        text: "Do you want to resume playback of %1 at %2?".arg(item.title).arg(item.resumeString)
```

Comment:
Can we have two buttons just like Kodi has?

"Resume at %1"
"Start from beginning"